sim/warrior/arms/mortal_strike.go

```diff
@@ -0,0 +1,49 @@
+package arms
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/warrior"
+)
+
+func (war *ArmsWarrior) RegisterMortalStrike() {
+	war.mortalStrike = war.RegisterSpell(core.SpellConfig{
+		ActionID:       core.ActionID{SpellID: 12294},
+		SpellSchool:    core.SpellSchoolPhysical,
+		ProcMask:       core.ProcMaskMeleeMHSpecial,
+		Flags:          core.SpellFlagAPL | core.SpellFlagIncludeTargetBonusDamage | core.SpellFlagMeleeMetrics,
+		ClassSpellMask: warrior.SpellMaskMortalStrike | warrior.SpellMaskSpecialAttack,
+
+		RageCost: core.RageCostOptions{
+			Cost:   20,
+			Refund: 0.8,
+		},
+
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: core.GCDDefault,
+			},
+			CD: core.Cooldown{
+				Timer:    war.NewTimer(),
+				Duration: time.Millisecond * 4500,
+			},
+		},
+
+		DamageMultiplier: 1.0,
+		CritMultiplier:   war.DefaultMeleeCritMultiplier(),
```

Comment:
Not sure if anyone ever is going to use this, but you are not setting the `ThreatMultiplier`

Reply:
Yeah, I'm not really sure what to set it to and I don't think we track threat for DPS sims